### PR TITLE
relval matrix to use HLT relval2016 

### PR DESCRIPTION
**Pull request by hengne**
Changes:
- relval 2015D data workflows to use HLT relval2016
- relval fastsim workflows to use HLT relval2016
- changes made on top of PR#14107
- backport fixes of fastsim from 81X PR #14098

### CODE REVIEW

**Comment by hengne** [comment]
please test

**Comment by hengne** [comment]
+1

**Comment by hengne** [comment]
backport fixes of fastsim from 81X PR #14098

**Comment by hengne** [comment]
please test

**Comment by hengne** [comment]
+1

2016年4月22日星期五，cmsbuild <notifications@github.com> 写道：

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -963,10 +963,15 @@ def identityFS(wf):
...
+hltKey2016='relval2016'
+menuR2_2016 = autoHLT[hltKey2016]
+steps['HLTDR2_2016']=merge( [ {'-s':'L1REPACK:Full,HLT:@%s'%hltKey2016,},{'--conditions':'auto:run2_hlt_relval'},{'--era' : 'Run2_2016'},steps['HLTD'] ] )
+
 
 # custom function to be put back once the CSC tracked/untracked will have been fixed.. :-)
-steps['RECODR2_50ns']=merge([{'--scenario':'pp','--conditions':'auto:run2_data_relval','--customise':'Configuration/DataProcessing/RecoTLR.customiseDataRun2Common',},dataReco])
-steps['RECODR2_25ns']=merge([{'--scenario':'pp','--conditions':'auto:run2_data_relval','--customise':'Configuration/DataProcessing/RecoTLR.customiseDataRun2Common_25ns',},dataReco])
+steps['RECODR2_50ns']=merge([{'--scenario':'pp','--conditions':'auto:run2_data_relval','--era':'Run2_50ns',},dataReco])
+steps['RECODR2_25ns']=merge([{'--scenario':'pp','--conditions':'auto:run2_data_relval','--era':'Run2_25ns',},dataReco])
```
Comment (slava77) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: I missed the changes made here, sorry.
We can't drop Run2_25ns fully, there should still be a customisation kept for 25ns dataConfiguration/DataProcessing/RecoTLR.customisePostEra_Run2_25ns is needed
We are missing HCAL flags settings special only for 25ns data.
For 2016 data processing there is a similar modification needed

Comment (hengne) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: @slava77 Thanks!  updated in 80x PR #14342 . 81x version is ongoing.

Comment (hengne) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: @slava77 also put there for 81x PR #14342 14343

**Pull request merged**